Cargo.toml

```diff
@@ -648,7 +652,7 @@ walkdir = "2.3.3"
 warp = { version = "0.3.5", features = ["tls"] }
 warp-reverse-proxy = "1.0.0"
 which = "4.2.5"
-x25519-dalek = "1.2.0"
+x25519-dalek = { git = "https://github.com/aptos-labs/x25519-dalek" }
```

Comment:
why are we forking dalek? how do we plan to maintain it current with upstream?

Reply:
For context, the upstream branch has a very old `zeroize` dependency. Many people have had to fork in order to make it work with newer dependencies (see this issue i created in 2022 before starting Aptos for example -> https://github.com/dalek-cryptography/curve25519-dalek/issues/388). I think we can pretty easily sync changes with the upstream but will let @mstraka100 or @alinush chime in here

Reply:
have you considered submitting a PR upstream to update zeroize?

Reply:
There actually is an update to x25519-dalek (2.0.0-rc.3) that updates zeroize, but it's still a release candidate and has not been finalized yet

Reply:
2.0.0 seems stable now: https://docs.rs/x25519-dalek/2.0.0/x25519_dalek/

Reply:
I believe there are other parts of our codebase that use our dalek fork anyways, so I'd propose updating it in a separate PR if we do update it